timesketch/lib/analyzers/yetiindicators.py

```diff
@@ -23,6 +24,21 @@
     "attack-pattern": "HIGH_VOLTAGE",
 }
 
+# Maps Yeti indicator locations to a Timesketch intelligence type
+INDICATOR_LOCATION_MAPPING = {"filesystem": "fs_path"}
+
+
+# Maps Yeti observable types to Timesketch observable types
+OBSERVABLE_INTEL_MAPPING = {
```

Comment:
This mapping looks like something that might change in the future. Would it make sense to have this mapping as part of the config instead of hardcoding it here?

Reply:
I don't think so, at least not for now. These values are set in both codebases, and won't change from deployment to deployment, unless they fork Yeti and add new observable types that aren't in the main repo.